tests/common/test_transaction.py

```diff
@@ -1014,16 +989,15 @@ def test_create_transfer_transaction_single_io(tx, user_pub, user2_pub,
     assert transfer_tx.fulfillments_valid([tx.conditions[0]]) is True
 
 
-@mark.skip(reason='FIXME: When divisible assets land')
 def test_create_transfer_transaction_multiple_io(user_pub, user_priv,
```

Comment:
The transfer transaction still has 2 inputs and two outputs (the only difference is that the two inputs come from 2 conditions of the same transaction)